packages/legacy-scripting-runner/test/example-app/index.js

```diff
@@ -313,7 +313,7 @@ const legacyScriptingSource = `
       },
 
       movie_pre_poll_env_var: function(bundle) {
-        bundle.request.url = 'https://{{process.env.SECRET_HTTPBIN_URL}}/get';
+        bundle.request.url = '{{process.env.SECRET_HTTPBIN_URL}}/get';
```

Comment:
I did this so we can spin up httpbin and auth-json-server locally and run the tests really fast:

```
$ AUTH_JSON_SERVER=http://localhost:3000 HTTPBIN_URL=http://localhost:3001 yarn test
```

Reply:
Given this `HTTPBIN_URL=http://localhost:3001,` it seems a need to rename the variable to something more generic.